include/RAJA/exec-sequential/scan_sequential.hxx

```diff
@@ -69,6 +69,12 @@ namespace detail
 namespace scan
 {
 
+RAJA_INLINE
```

Comment:
Moved this here to avoid multiple definitions in openmp and agency. Ideally this would go into a .cxx file somewhere, but there isn't one it fits well into.